Lighthouse2/Lighthouse2/Lighthouse2.csproj

```
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
```

Add default namespace and assembly name
```suggestion
    <PropertyGroup>
        <RootNamespace>LBPUnion.Lighthouse2</RootNamespace>
        <AssemblyName>LBPUnion.Lighthouse2</AssemblyName>
    </PropertyGroup>

```